application/controllers/QuestionAdministrationController.php

```diff
@@ -336,7 +336,7 @@ public function actionListQuestions($surveyid, $landOnSideMenuTab = 'settings')
         $questionGroupService = $diContainer->get(
             LimeSurvey\Models\Services\QuestionGroupService::class
         );
-        $aData['groupModel'] = $questionGroupService->getGroupData($iSurveyID);
+        $aData['groupModel'] = $questionGroupService->getGroupData($iSurveyID, $aData['oSurvey']);
```

Comment:
Hm why would you need both survey id and the survey model? Survey model contains id.

Reply:
Ah, that's an unnecessary leftover...